Dockerfile

```diff
@@ -66,69 +117,18 @@ RUN wget -q -O  -  https://sh.rustup.rs | \
   RUSTUP_HOME=/usr CARGO_HOME=/usr \
   sh -s -- -y --profile minimal --no-modify-path
 
-# Get Chrome and Firefox (borrowed from https://github.com/SeleniumHQ/docker-selenium)
-
-ARG CHROME_VERSION="latest"
-ARG FIREFOX_VERSION="latest"
-# Note: geckodriver version needs to be updated manually
-ARG GECKODRIVER_VERSION="0.34.0"
+COPY --from=selenium-manager-image /opt/firefox /opt/firefox
+COPY --from=selenium-manager-image /opt/geckodriver /opt/geckodriver
```

Comment:
Could you try merging these COPY commands into one? I guess it will reduce the number of layers in the Docker image, making the build faster.